Add Troubleshooting - Date values are parsed or serialized incorrectly (#3093)
DevExpress/devextreme-documentation: concepts/70 Data Binding/90 Troubleshooting/Date values are parsed or serialized incorrectly.md

**Issue**: Date values have an unexpected format when they are displayed in the component or sent to the server.

**Solution**:     

> Different browsers parse date strings differently. To compensate for this inconsistency, DevExtreme components use an internal parser. It only supports date strings that comply with the <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">ISO 8601</a> standard, that is, strings that have the following formats:

Reviewer: ```suggestion
Different browsers parse date strings differently. To compensate for this inconsistency, DevExtreme components use an internal parser. The latter only supports date strings that comply with the <a href="https://en.wikipedia.org/wiki/ISO_8601" target="_blank">ISO 8601</a> standard, that is - strings that have the following formats:
```

- `"yyyy-MM-dd"` - parsed as a local date  

- `"yyyy-MM-ddTHH:mm:ss"` - parsed as local date and time  

- `"yyyy-MM-ddTHH:mm:ssZ"` - parsed as UTC date and time  

- `"yyyy-MM-ddTHH:mm:ssx"` - parsed as date and time with a timezone

> Date strings in other formats are passed to the Date constructor. In some instances, this approach produces different results. Refer to the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse" target="_blank">Date.parse()</a> article for more information. We encourage you to use the ISO 8601 formats to guarantee the absence of possible issues.

Reviewer: ```suggestion
Date strings in other formats are passed to the Date constructor. There are times when this approach produces different results. Refer to the <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse" target="_blank">Date.parse()</a> article for more information. We encourage you to use the ISO 8601 formats to guarantee the absence of possible issues.
```
